docs/howto/troubleshoot/logs.md

```diff
@@ -88,15 +90,26 @@ errors can be frustrating!
 
 #### Look at dask-gateway logs
 
-The following query will show logs from all the components of dask-gateway
-infrastructure - the controller, api and the proxy. Note that this does *not*
-show logs from specific schedulers or worker pods a user might have started.
+The following query will show logs from all the components of dask-gateway -
+the controller, API, proxy, individual schedulers and workers.
 
 ```
 labels.k8s-pod/app_kubernetes_io/name="dask-gateway"
 resource.labels.namespace_name="<namespace>"
 ```
 
+To look at just the *scheduler* and *worker* logs of a specific user, you can
+try:
+
+```
+labels.k8s-pod/app_kubernetes_io/name="dask-gateway"
+resource.labels.namespace_name="<namespace>"
```

Comment:
I am not entirely sure what `namespace` is in the context of these queries. Adding a concrete example might help somebody like me understand this quicker.

Reply:
good point, @jbusecke! For quick reference in most cases this is 'prod', and the value is the 'name' for the hub assigned here: https://github.com/2i2c-org/infrastructure/blob/a4aefed81700fcbe2a6224b30a61e8abd155ba76/config/clusters/leap/cluster.yaml#L23.

I'll find a way to document this and how users can determine it

Reply:
Awesome thanks